server/models/video/timecode-thumbnail-manifest.ts

```diff
@@ -0,0 +1,90 @@
+import { join } from 'path'
+import * as uuidv4 from 'uuid/v4'
+import { AfterDestroy, AllowNull, BelongsTo, Column, CreatedAt, ForeignKey, Model, Table, UpdatedAt } from 'sequelize-typescript'
+import { STATIC_PATHS, WEBSERVER } from '../../initializers/constants'
+import { logger } from '../../helpers/logger'
+import { remove } from 'fs-extra'
+import { CONFIG } from '../../initializers/config'
+import { VideoModel } from './video'
+
+/**
+ * Used to store timecode thumbnail manifests (.vtt files listing thumbnails used
+ * for video scrubbing previews)
+ *
+ * A video has zero or one manifest at most.
+ */
+@Table({
+  tableName: 'timecodeThumbnailManifest',
+  indexes: [
+    {
+      fields: [ 'videoId' ]
+    }
+  ]
+})
+export class TimecodeThumbnailManifestModel extends Model<TimecodeThumbnailManifestModel> {
```

Comment:
I think you should store how many thumbnails you generated

Reply:
Done in the new version.